src/RecordManager/Base/Command/Records/Export.php

```diff
@@ -183,6 +218,78 @@ protected function configure()
             );
     }
 
+    /**
+     * Collect range and source parameters from user-provided options.
+     *
+     * @return array
+     */
+    protected function gatherRangeAndSourceParams(): array
+    {
+        $params = [];
+        if ($this->fromDate && $this->untilDate) {
+            $params['$and'] = [
+                [
+                    'updated' => [
+                        '$gte'
+                            => $this->db->getTimestamp(strtotime($this->fromDate)),
+                    ],
+                ],
+                [
+                    'updated' => [
+                        '$lte'
+                            => $this->db->getTimestamp(strtotime($this->untilDate)),
+                    ],
+                ],
+            ];
+        } elseif ($this->fromDate) {
+            $params['updated']
+                = ['$gte' => $this->db->getTimestamp(strtotime($this->fromDate))];
+        } elseif ($this->untilDate) {
+            $params['updated']
+                = ['$lte' => $this->db->getTimestamp(strtotime($this->untilDate))];
+        }
+        if ($this->fromCreateDate && $this->untilCreateDate) {
+            $params['$and'] = [
+                [
+                    'created' => [
+                        '$gte' => $this->db->getTimestamp(
+                            strtotime($this->fromCreateDate)
+                        ),
+                    ],
+                ],
+                [
+                    'created' => [
+                        '$lte' => $this->db->getTimestamp(
+                            strtotime($this->untilCreateDate)
+                        ),
+                    ],
+                ],
+            ];
+        } elseif ($this->fromCreateDate) {
+            $params['created'] = [
+                '$gte' => $this->db->getTimestamp(strtotime($this->fromCreateDate)),
+            ];
+        } elseif ($this->untilDate) {
```

Comment:
Oops, there's an old mistake here. Can we fix it (change to `$this->untilCreateDate`) while at it?

Reply:
Done!